### ISSUE TITLE
what does "showonce" attribute means

### ISSUE THREAD

**Issue by croban** [umbraco/UmbracoDocs#1538]
***This is the page with issues: https://github.com/umbraco/UmbracoDocs/edit/master/Reference/Config/dashboard/index.md***

**Comment by sofietoft** [comment]
Hi @croban 

I'm not entirely sure I understand what you're asking here.
Have you found an issue with the article? 🤔

/Sofie

**Comment by dampee** [comment]
@sofietoft Pretty sure he is talking about the "showOnce" attribute on the second line 

```xml
<tab caption="Welcome">
  <control showOnce="true" addPanel="true" panelCaption="">
        views/dashboard/settings/settingsdashboardintro.html
  </control>
</tab>
```

**Comment by dampee** [comment]
To be honest, I think it is not used anymore. I can't even remember what it does. Maybe @marcemarc  or @Jeavon  know.  :-o

**Comment by dampee** [comment]
@croban I can confirm that this property is obsolete in v7.  If you have an old version, this would make a particular to be shown only once (pre v7).

I prepared a small change ready to be published: b8f3e3d9248eccf407b44bc8c3d458c62505ee4a

Documentation will be up to date soon.  So i'll close this issue.
